example/run.bash

```
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+log() { echo ">>> $@"; }
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)